configs/openmrs_config/ampathforms/NCD_Consultation.json

```
@@ -279,6 +279,200 @@
 							"validators": []
 						}
 					]
+				},
+				{
+					"isExpanded": "true",
+					"label": "Right Eye Retinal Exam",
+					"questions": [
+						{
+							"id": "Creatinine",
```

I think we could have all our label id's `camelCased` starting with a small letter. This could be unified across all our forms. cc: @rbuisson

---

@kdaud  This is for a section Label not an ID, the way you put it there is the way you get it in the form

---

We could have the label ID's camelCased eg

```suggestion
							"id": "creatinine",
```
```suggestion
							"id": "creatinineClearance",
```

Once we have the id labels unified across all the forms; it will ease locating page elements when automating QA scenarios a head of us

---

Thanks @kdaud for this and I have noted it. That ID however is a single word